### PR TITLE
Fix cypress

### PR DESCRIPTION
**Pull request by karolina-siemieniuk-morawska**

### CODE REVIEW

**Review by MJedr — APPROVED** [review]
you have to rebase ;

File: .github/workflows/e2e-tests.yml [hunk 1]
```diff
@@ -189,7 +189,7 @@ jobs:
         id: setup-chrome
         uses: browser-actions/setup-chrome@v1
         with:
-          chrome-version: '114'
+          chrome-version: '999988'
```
Comment (drjova) [inline .github/workflows/e2e-tests.yml:192]: what's that? :)

Comment (MJedr) [reply .github/workflows/e2e-tests.yml:192]: chromium by commit position https://github.com/browser-actions/setup-chrome#parameters

Comment (karolina-siemieniuk-morawska) [reply .github/workflows/e2e-tests.yml:192]: With this action [you can only specify chrome version as stable, beta, dev, and canary](https://github.com/browser-actions/setup-chrome#setup-chrome), directly specifying chrome version is not implemented yet. But you can specify chromium commit number which represents version, so I chose this one.

**Pull request merged**